docs/en-US/component/scrollbar.md

```diff
@@ -60,6 +60,7 @@ scrollbar/manual-scroll
 | role ^(2.4.0) ^(a11y)             | role of view                                                                                                                    | ^[string]                                                           | —       |
 | aria-label ^(2.4.0) ^(a11y)       | aria-label of view                                                                                                              | ^[string]                                                           | —       |
 | aria-orientation ^(2.4.0) ^(a11y) | aria-orientation of view                                                                                                        | ^[enum]`'horizontal' \| 'vertical'`                                 | —       |
+| tabindex                          | tabindex of wrap container                                                                                                      | ^[number] / ^[string]                                               | —       |
```

Comment:
Perhaps we could add a version tag to indicate which versions support this configuration.
```suggestion
| tabindex ^(2.8.3)                 | tabindex of wrap container                                                                                                      | ^[number] / ^[string]                                               | —       |
```

Reply:
done